### PR TITLE
AA - Update auth forms with new design + add clear input icon

### PR DESCRIPTION
**Pull request by AntonovAtanas**
- Update UI inputs for login, register user, and register organization
- Add a clear input button
- Change colors on input active state
- Merge User and Organizator interfaces
- Update Event interface

- TODO: Validation and change input colors on error and success state
- TODO: Auth services

image

image

### CODE REVIEW

File: src/shared/interfaces/User.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
 export interface User {
   email: string;
-  role?: string;
+  organizatorName?: string;
+  role: string;
   firstName?: string;
   lastName?: string;
   region?: string;
+  address?: string;
```
Comment (Getsov) [inline src/shared/interfaces/User.ts]: Comment out address

File: src/shared/interfaces/User.ts [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+  phone?: string;
+  createdEvents: { createdEventsId: string }[];
```
Comment (Getsov) [inline src/shared/interfaces/User.ts]: optional

File: src/shared/interfaces/User.ts (hunk 1 above)
Comment (AntonovAtanas) [reply src/shared/interfaces/User.ts]: commented out address

File: src/shared/interfaces/User.ts (hunk 2 above)
Comment (AntonovAtanas) [reply src/shared/interfaces/User.ts]: edit to optional createdEvents

**Review by Getsov — APPROVED** [review]

**Pull request merged**